### PR TITLE
Add `describe instance type offering` permission to the installer role

### PR DESCRIPTION
**Pull request by oriAdler**
Add the permission to enable the installer role to list the supported
machine types in the region.

### What type of PR is this?
Bug.

### What this PR does / why we need it?
To fetch the supported machine types in a region via the installer role.

### Which Jira/Github issue(s) this PR fixes?
Related: [SDA-6357](https://issues.redhat.com/browse/SDA-6357)

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by oriAdler** [comment]
@pvasant PTAL

**Comment by dofinn** [comment]
signed off in linked card by PM
/approve
/lgtm
/hold

Please unhold when ready to commit  @pvasant

**Comment by pvasant** [comment]
@dofinn we have PM approval in the jira https://issues.redhat.com/browse/SDA-6357

**Comment by oriAdler** [comment]
@dofinn I've added the permission to the `ocm role` as well.
@arendej approved these changes in https://issues.redhat.com/browse/SDA-6357.

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**
